lib/backend/leveldb.js

```diff
@@ -763,7 +762,6 @@ ConsistentHash.prototype.addData = function addData(vnode, data, cb) {
 
                 return _cb(err);
             });
-            console.log('{"%s":"%s"}', vnode, data);
```

Comment:
This might be relevant output for `./bin/fash.js add-data`?

Reply:
agreed. will add back.